devfile.yaml

```diff
@@ -0,0 +1,37 @@
+schemaVersion: 2.2.0
+metadata:
+  name: fleet-manager
+  version: 0.1.0
+  provider: Red Hat
+  icon: https://design.jboss.org/quarkus/logo/final/SVG/quarkus_icon_rgb_default.svg
```

Comment:
We have our logos btw https://design.jboss.org/apicurio/

Reply:
Thanks. Updated the url to use Apicurio Logo